src/analysis/complex/real_deriv.lean

```diff
@@ -79,6 +79,55 @@ theorem times_cont_diff.real_of_complex {n : with_top ℕ} (h : times_cont_diff
 times_cont_diff_iff_times_cont_diff_at.2 $ λ x,
   h.times_cont_diff_at.real_of_complex
 
+variables {E : Type*} [normed_group E] [normed_space ℂ E]
+
+lemma complex.restrict_scalars_one_smul_right' (x : E) :
+  continuous_linear_map.restrict_scalars ℝ ((1 : ℂ →L[ℂ] ℂ).smul_right x : ℂ →L[ℂ] E) =
+    re_clm.smul_right x + I • im_clm.smul_right x :=
+by { ext ⟨a, b⟩, simp [mk_eq_add_mul_I, add_smul, mul_smul, smul_comm I] }
+
+lemma complex.restrict_scalars_one_smul_right (x : ℂ) :
+  continuous_linear_map.restrict_scalars ℝ ((1 : ℂ →L[ℂ] ℂ).smul_right x : ℂ →L[ℂ] ℂ) = x • 1 :=
+begin
+  ext1 z,
+  dsimp,
+  apply mul_comm
+end
```

Comment:
Is there a more appropriate place for these two?

Reply:
`analysis.complex.basic`?

Reply:
If it doesn't need new imports, that makes sense to me.